cl/opinion_page/templates/includes/de_list.html

```diff
@@ -8,21 +8,42 @@
       <p class="hidden-xs">Document Number</p>
     </div>
     <div class="col-xs-3 col-sm-2">Date&nbsp;Filed</div>
-    <div class="col-xs-8 col-sm-6">
-      <div class="col-xs-9 description-header">
+    <div class="col-xs-8 col-sm-2">
+      <div class="col-xs-3 description-header">
         Description
       </div>
-      <div class="col-xs-3">
-        <a href="{% url 'view_download_docket' docket.id %}" class="btn export-csv visible-xs">
+      <div class="col-xs-6 description-header flex justify-content-end">
+        <span id="mobile-export-csv-error" class="hidden visible-xs text-danger">
+          <i class="fa fa-info-circle"></i>&nbspTry again later.
+        </span>
+      </div>
+      <div class="col-xs-3 flex">
+        <a hx-get="{% url 'view_download_docket' docket.id %}"
+          hx-swap="none" hx-indicator="#mobile-spinner"
+          hx-trigger="click" class="btn export-csv visible-xs">
           <div class="flex align-items-center">
             <i class="fa fa-download gray"></i>&nbsp;
             <span>CSV</span>
           </div>
         </a>
+        &nbsp;
+        <span>
+          <i id="mobile-spinner" class="htmx-indicator fa fa-spinner fa-spin fa-lg"></i>
+        </span>
       </div>
     </div>
-    <div class="flex justify-content-end col-xs-3">
-      <a href="{% url 'view_download_docket' docket.id %}" class="btn btn-default hidden-xs export-csv">
+    <div class="flex justify-content-end col-xs-7 hidden-xs">
```

Comment:
Should this have a `sm` class, instead of an `xs` class?

Reply:
You're right. This change won't affect the responsive design but it will make the code's intent clearer for other developers. Thanks!